testcases/gui-self-cluster-check.md

```diff
@@ -0,0 +1,33 @@
+# Running Selenium cluster test cases for DOP
+<b>tcid:</b> gacc01 <br>
+<b>name:</b> "Running Selenium cluster test cases for DOP"<br>
```

Comment:
"Running Selenium cluster test cases for DOP" 
to "DOP Self Connected Cluster Check"